styles.css

```diff
@@ -2,10 +2,64 @@
     box-sizing: border-box;
 }
 
-body, html {
+body, html, a {
     font-family: 'Poppins', sans-serif;
     margin: 0;
     padding: 0;
     color: #293241;
     font-size: 18px;
+    text-decoration: none;
```

Comment:
čia `text-decoration` netinka, nes dabar jis galioja ir `body` ir `html`. Jis turėtų galioti tik visiems `a` elementams, nepriklausomai nuo to, kokia jo būsena (:link, :hover ir t.t.)